index.css

```diff
@@ -1,15 +1,41 @@
+/*Start Default block  */
 .body{
     margin: 0;
 }
 
 p{
     margin: 0;
 }
+/*End Default block  */
 
+/*Start Page block */
```

Comment:
I think we should have .page{} at this location. it was empty for now, someone need it, they can modify.
Otherwise, someone may add this later in wrong order